src/ProjectTemplates/Web.ItemTemplates/content/RazorView/.template.config/localize/templatestrings.en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "author": "Microsoft",
+  "name": "Razor View",
+  "description": "A Razor view without a page model",
+  "postActions/openInEditor/description": "Opens Index.cshtml in the editor"
```

Comment:
Similar comment regarding `Index.cshtml` as to the HomeController one. Maybe it's better to state "Opens the generated view in the editor"

Reply:
kept it `Opens created view in the editor` to keep it consistent with `Opens created controller in the editor` for the controller scenarios.

Reply:
looks good @sayedihashimi ?